### ISSUE TITLE
Starting V3.0 doesn't show any GUI

### ISSUE THREAD

**Issue by riQQ** [Manuel-777/MTG-Arena-Tool#836]
Starting V3.0 doesn't show any GUI, only the two MTGA Tool processes are running.

**Comment by TroubiCZ** [comment]
+1

**Comment by KerfuffleV2** [comment]
Same problem. Based on the console messages, it appears to be having a problem loading its local DB:

```linux
Local DB: Loading data...
(node:30221) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'send' of undefined
    at ipcSend (/tmp/.mount_MTG-ArMBDqkm/resources/app.asar/lib/shared/db/databaseUtil.js:40:25)
    at showBusy (/tmp/.mount_MTG-ArMBDqkm/resources/app.asar/lib/shared/db/databaseUtil.js:52:3)
    at ElectronStoreDatabase.find (/tmp/.mount_MTG-ArMBDqkm/resources/app.asar/lib/shared/db/ElectronStoreDatabase.js:131:32)
    at MigrationDatabase.find (/tmp/.mount_MTG-ArMBDqkm/resources/app.asar/lib/shared/db/MigrationDatabase.js:61:29)
    at startUpdater (/tmp/.mount_MTG-ArMBDqkm/resources/app.asar/lib/main.js:91:24)
    at App.<anonymous> (/tmp/.mount_MTG-ArMBDqkm/resources/app.asar/lib/main.js:71:5)
    at App.emit (events.js:199:15)
(node:30221) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:30221) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

**Comment by riQQ** [comment]
Installing 3.0.1 fixed it for me. The fix was implemented in a72f583660916cb4598c90f892a2b45228e274e4.
